server/datastore/mysql/migrations/tables/migration_test.go

```diff
@@ -0,0 +1,83 @@
+// Package tables holds fleet table migrations.
+//
+// Migrations can be tested with tests following the following format:
+//
+//	$ cat 20220208144831_AddSoftwareReleaseArchVendorColumns_test.go
+//
+//	[...]
+//	func TestUp_20220208144831(t *testing.T) {
```

Comment:
Would it be very complex to make `make migration` generate that file automatically in addition to the migration file?

Reply:
Good idea, I'll give it a go.

Reply:
https://github.com/fleetdm/goose/pull/1